kong/pdk/response.lua

```diff
@@ -110,6 +118,59 @@ local function new(self, major_version)
     [16] = "Unauthenticated",
   }
 
+  local HTTP_MESSAGES = {
```

Comment:
Because this is generic `error` method we cannot just have selected error codes, but we need to have them all here. The error handler only handled selected codes, this should handle them all.